jigsaw-core.php

```diff
@@ -101,7 +101,7 @@
         return new BladeMarkdownEngine($compilerEngine, $c[FrontMatterParser::class]);
     });
 
-    (new BladeDirectivesFile($c['cwd'] . '/blade.php'))->register($bladeCompiler);
+    BladeDirectivesFile::init($c['cwd'] . '/blade.php', $bladeCompiler)->register();
```

Comment:
I think we can ditch the `init` static function, and just `new` the class up here directly.